values.yaml

```diff
@@ -0,0 +1,44 @@
+# Default values for infinispan-helm-charts.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image: quay.io/infinispan/server:12.1.3
+initContainerImage: registry.access.redhat.com/ubi8-micro
+
+replicas: 1
+
+service:
+  type: DataGrid
+  container:
+    storage: 1Gi
+    storageClassName: ""
+    ephemeralStorage: false
+  resources:
+    extraJvmOpts: ""
+    cpu: 500m
+    memory: 512Mi
+
+security:
+  endpointAuthentication: true
+  endpointSecretName: ""
+  endpointSecretPassword: ""
```

Comment:
I know these values are the same as the Infinispan CR spec, but as the helm-charts are a distinct entity maybe we should re-evaluate naming here?

IMO below is much simpler:

```yaml
security:
  authentication: true
  secretName: ""
  secretPassword ""
```

WDYT @dmvolod @rigazilla ?

Reply:
Agree, @ryanemerson . We can to start design new structure for both operator and helm chart and sync them between each others.

Reply:
> We can to start design new structure for both operator and helm chart and sync them between each others.

I think someone common terminology between the two (and Infinispan server) is a good thing, but IMO we shouldn't tightly couple the operator and the charts. For example, I wouldn't want a change required in the operator to hold back helm-chart development and vice-versa.

Reply:
Agree, structure a bit refactored